docs/tutorials/tutorial-1.rst

```diff
@@ -134,6 +134,14 @@ Edit the ``setup.py`` file to include a package helper for Django:
       }
   )
 
+Building Django has Javascript dependencies that are installed with NPM.
+Pick the appropriate command to install NPM on your OS, or visit the `NPM website <https://www.npmjs.com/>`_ and follow the instructions.
+
+.. code-block:: bash
+
+  $ brew install npm  # OSX with Homebrew
```

Comment:
If you're going to recommend `brew` (which I think is good), please link to the brew website in some fashion, as we want to be as beginner-friendly as possible.